edb/lib/std/25-setoperators.edgeql

```diff
@@ -92,7 +92,7 @@ std::`??` (l: OPTIONAL anytype, r: SET OF anytype) -> SET OF anytype {
 
 
 CREATE TERNARY OPERATOR
-std::`IF` (condition: bool, if_true: SET OF anytype,
+std::`IF` (if_true: SET OF anytype, condition: bool,
```

Comment:
Why do you think this is necessary or important?

Reply:
Because the error message ends up listing the signature types in an odd order, which confused me when I tried `SELECT 1 IF True ELSE False` since the error message complained about operands being `std::bool`, `std::int64`, `std::bool`. This makes it look like there's something wrong with the error message.

Reply:
OK, makes sense.